cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -117,7 +117,8 @@ func newHcsStandaloneTask(ctx context.Context, events publisher, req *task.Creat
 }
 
 // createContainer is a generic call to return either a process/hypervisor isolated container, or a job container
-//  based on what is set in the OCI spec.
+//
+//	based on what is set in the OCI spec.
```

Comment:
This was turned into a command, since the original text was indented by two spaces instead of one.
Should be just:
```
// based on what is set in the OCI spec.
```